source/functions/initializeWecSim.m

```diff
@@ -251,8 +258,8 @@
 if ~isempty(idx)
     for kk = 1:length(idx)
         it = idx(kk);
-        body(it).hydroForcePre(waves.omega,waves.direction,simu.cicTime,waves.bem.count,simu.dt,...
-            simu.rho,simu.gravity,waves.type,waves.waveAmpTime,simu.stateSpace,simu.b2b);
+        body(it).hydroForcePre(waves(1).omega,waves(1).direction,simu.cicTime,waves(1).bem.count,simu.dt,...
+            simu.rho,simu.gravity,waves(1).type,waves(1).waveAmpTime,simu.stateSpace,simu.b2b); % TODO need to rethink how user defined exc is implemented for multiple waves
```

Comment:
@salhus Minor comment: can you remove the comment and add as a project board card. That should be relatively straightforward to allow user defined waves but will not prioritize it until next fiscal year.

Reply:
- done